.storybook/manager-head.html

```diff
@@ -0,0 +1 @@
+<link rel="shortcut icon" href="/favicon-%NODE_ENV%.ico" />
```

Comment:
Nit: Self-closing tag not necessary in HTML5?

Reply:
Looks like Prettier is adding it back in. I'll see if I can disable that.

Reply:
Holy wow, this turns out to be a _contentious_ topic in the Prettier community: https://github.com/prettier/prettier/issues/5246

TL;DR: There's no option to disable it right now, and Prettier will always add the self-closing tag slash back in.